docs/about/authors.rst

```diff
@@ -34,3 +34,4 @@ The following developers have contributed to the PyMeasure package:
 | Benjamin Klebel-Knobloch
 | Hud Wahab
 | Nicola Corna
+| Domenic Prete
```

Comment:
`AUTHORS.txt` mentions also two other name. Is there a reason to not add them in `authors.rst` ? I believe the two files should be in sync.

Reply:
Yes most probably I messed up somewhere (sigh...) in syncing everything, in order to address this may I insert in authors.rst the names present in AUTHORS.TXT ? Is that a valid approach ?

Reply:
at some point we should unify this so we only need 1 file, and the docs generator automatically pulls AUTHORS.txt (cf. #341)